best-practices.md

```diff
@@ -274,6 +275,9 @@ ended up doing. Following these recommendations makes for more legible catalogs.
 5. Items should be stored in subdirectories of their parent catalog. 
 This means that each item and its assets are contained in a unique subdirectory.
 6. Limit the number of items in a catalog or sub-catalog, grouping / partitioning as relevant to the dataset.
+7. In a nested hierarchy of catalogs and collections make the different levels 'homogeneous' - all catalogs or all collections. 
```

Comment:
That sounds a bit like there should be only one level of collections, although it should also be fine to have e.g. all levels being collections.

Reply:
Cool, I'll try again. I was thinking about that as I was writing it but didn't manage to find a way to succinctly get it across. This was weirdly more challenging to explain than I was expecting.